plugins/source/k8s/resources/services/apps/daemon_sets.go

```diff
@@ -98,10 +98,10 @@ func DaemonSets() *schema.Table {
 				IgnoreInTests: true,
 			},
 			{
-				Name:        "cluster_name",
-				Description: "The name of the cluster which the object belongs to. This is used to distinguish resources with same name and namespace in different clusters. This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.",
+				Name:        "zzz_cluster_name",
```

Comment:
This field was deprecated https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta.
We didn't catch it as tests were not running for this plugin